tests/Unit/PointwiseFunctions/MathFunctions/Test_Sinusoid.cpp

```diff
@@ -44,12 +44,12 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.MathFunctions.Sinusoid",
     CHECK(sinusoid.first_deriv(one)[s] == approx(mapped_first_deriv[s]));
     CHECK(sinusoid.second_deriv(one)[s] == approx(mapped_second_deriv[s]));
   }
-  test_math_helpers::test_pup_function(sinusoid);
-
-  // Test base class serialization
-  Parallel::register_derived_classes_with_charm<MathFunction<1>>();
-  test_math_helpers::test_pup_function(std::unique_ptr<MathFunction<1>>{
```

Comment:
I believe this removes all the uses of `test_pup_function` (both versions), so it should remove the function as well.